Tasks/Maven/CodeAnalysis/Common/FindbugsTool.ts

```diff
@@ -100,6 +96,55 @@ export class FindbugsTool extends BaseTool {
             tl.debug(`[CA] A FindBugs report was found for for module '${moduleName}' containing ${violationCount} issues - ${xmlReport}`);
         });
 
+        return jsonCounts;
+    }
+
+    /**
+     * Extracts the number of affected files and the number of violations from a report JSON object
+     * @param data JSON object to parse
+     * @returns a tuple of [affected_file_count, violation_count]
+     */
+    private static parseJson(data:any):[number, number] {
+        var fileCount:number = 0;
+        var violationCount:number = 0;
+
+        // If available, FileStats is much easier to fetch data from than ClassStats.
+        if (!!data.BugCollection.FindBugsSummary[0].FileStats) {
+            data.BugCollection.FindBugsSummary[0].FileStats.forEach((file: any) => {
+                if (file.$.bugCount > 0) {
+                    fileCount++;
+                    violationCount += Number(file.$.bugCount);
+                }
+            });
+        } else if (!!data.BugCollection.FindBugsSummary[0].PackageStats &&
+            !!data.BugCollection.FindBugsSummary[0].PackageStats[0].ClassStats) {
```

Comment:
However, the Gradle FindBugs plugin lacks the FileStats section and thus a more involved solution is required.